### ISSUE TITLE
Integration tests fail

### ISSUE THREAD

**Issue by csperkins** [glasgow-ipl/ips-protodesc-code#18]
I get the following error running the integration tests
```
(ips-protodesc-code) [mangole] > make integrationtests
npt examples/draft-mcquistin-augmented-ascii-diagrams.xml -of rust
File [0]  --> /Users/csp/Projects/EPSRC-BrightIDEAS/repos/ips-protodesc-code/examples/draft-mcquistin-augmented-ascii-diagrams.xml
	/Users/csp/Projects/EPSRC-BrightIDEAS/repos/ips-protodesc-code/examples/output/draft/draft-mcquistin-augmented-ascii-diagrams/rust/src/main.rs
	/Users/csp/Projects/EPSRC-BrightIDEAS/repos/ips-protodesc-code/examples/output/draft/draft-mcquistin-augmented-ascii-diagrams/rust/Cargo.toml
make: *** No rule to make target 'examples/output/draft/draft-mcquistin-quic-augmented-diagrams/rust/', needed by 'integrationtests'.  Stop.
```

**Comment by djichthys** [comment]
Tested this on debian unstable. Works now.

**Comment by lumisota** [comment]
Fixed in de56480.
